app/src/main/java/com/sun/mangareader01/ui/main/MainActivity.kt

```diff
@@ -56,6 +59,10 @@ class MainActivity : FragmentActivity(),
         }
 
         override fun onChapterClick(chapter: Chapter?) {
+            Intent(baseContext, ReadActivity::class.java).apply {
```

Comment:
convention 2.11
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment